lib/workers/evm_server.rb

```diff
@@ -241,11 +241,7 @@ def start_workers
     #############################################################
     # Start all the configured workers
     #############################################################
-    @current_server.clean_heartbeat_files # Appliance specific
-    @current_server.sync_config
-    @current_server.start_drb_server
-    @current_server.sync_workers
-    @current_server.wait_for_started_workers
+    @current_server.start_workers
```

Comment:
nice catch. don`t think this was an obvious change until you probably saw all the extra delegation.